### PR TITLE
Add new board BastBLE

### PR DESCRIPTION
**Pull request by Soulkrown**
Hi
I would like to be able to add this new BastBLE card from electronic cats to your repository.
I await your feedback and response.
I thank you in advance.
Best regards

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -182,6 +182,7 @@ jobs:
         - "arduino_nano_33_iot"
         - "arduino_zero"
         - "bast_pro_mini_m0"
+	- "bastBLE"
```
Comment (tannewt) [inline .github/workflows/build.yml:185]:
```suggestion
        - "bastBLE"
```

**Review by tannewt — CHANGES_REQUESTED** [review]
Please follow up when the CI fails. Thanks! I don't see any other potential issues.

**Comment by tannewt** [comment]
I suggest installing pre-commit locally to test more rapidly: https://pre-commit.com/#installation

**Comment by tannewt** [comment]
The bastble build is broken. Looks like it needs a function name updated: https://github.com/adafruit/circuitpython/pull/3662/checks?check_run_id=1410247014

**Comment by sabas1080** [comment]
@tannewt ready, thanks!

File: ports/nrf/boards/bastble/pins.c [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    { MP_ROM_QSTR(MP_QSTR_MOSI), MP_ROM_PTR(&pin_P1_06) },
+    { MP_ROM_QSTR(MP_QSTR_MISO), MP_ROM_PTR(&pin_P0_15) },
+
+    { MP_ROM_QSTR(MP_QSTR_SCK), MP_ROM_PTR(&pin_P1_00) },
+    { MP_ROM_QSTR(MP_QSTR_LED), MP_ROM_PTR(&pin_P0_24)  },
+
+    { MP_ROM_QSTR(MP_QSTR_AREF), MP_ROM_PTR(&pin_P0_31) },
+
+    // voltage sense battery
+    { MP_ROM_QSTR(MP_QSTR_VBAT), MP_ROM_PTR(&pin_P0_26)  },
```
Comment (tannewt) [inline ports/nrf/boards/bastble/pins.c:37]: Note, most boards name this pin VOLTAGE_MONITOR or BATTERY.

**Review by tannewt — APPROVED** [review]
Looks good! One comment that you can make a follow up PR for if you like.

**Pull request merged**